src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+# WooCommerce
```

Comment:
Deze titel is nu hetzelfde als van de bestaande pagina, dus nu krijg je 2x "WooCommerce" in de sidebar. 
```suggestion
# WooCommerce (v5.0.0)
```